### PR TITLE
{lib}[GCCcore/12.3.0] orthAgogue v20141105, cmph v2.0, tbb v2020.3

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node373.hydra.os - Linux Rocky Linux 8.8, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/WilleBell/f40e02a280b1c384a8d685b1923ec212 for a full test report.

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/o/orthAgogue/orthAgogue-20141105-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+_commit = 'ebe3467'
+
+homepage = 'https://github.com/guyleonard/orthagogue'
+description = """
+orthAgogue: a tool for high speed estimation of homology relations within and between species
+in massive data sets. orthAgogue is easy to use and offers flexibility through a range of
+optional parameters.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/o/orthAgogue/orthAgogue-20141105-foss-2023a.eb:14]: looks like this could be moved down to `gompi`?

Comment (WilleBell) [reply easybuild/easyconfigs/o/orthAgogue/orthAgogue-20141105-foss-2023a.eb:14]: Yes, indeed. Thank you @migueldiascosta

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen3

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
Going in, thanks @WilleBell!

**Pull request merged**